Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/InvoluntaryProject/SearchSchool.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@page "/start-new-project/school-name"
+@model Dfe.PrepareConversions.Pages.InvoluntaryProject.SearchSchoolModel
+
+@{
+    Layout = "_Layout";
+    ViewBag.Title = "What is the school name - Manage an academy conversion";
```

Comment:
I may be misremembering but isn't the "- Manage an academy conversions" part is concatenated somewhere else?

Reply:
Just going off what the story has asked for. Might show up for screen readers I suppose.
![image](https://user-images.githubusercontent.com/109524605/216584521-2c83d6bb-91de-4a0c-af77-09906269c4de.png)